assets/config/0.5.5.1-coins.json

```diff
@@ -10338,5 +10338,55 @@
     "active": false,
     "is_segwit_on": false,
     "currently_enabled": false
+  },
+"ACTN": {
+    "coin": "ACTN",
+    "name": "Action Coin",
+    "type": "Smart Chain",
+    "coinpaprika_id": "actn-action-coin",
+    "coingecko_id": "action-coin",
+    "nomics_id": "actn-action-coin",
```

Comment:
nomics_id should be `ACTN`, taken from https://nomics.com/assets/actn-action-coin/widget (the `data-base="ACTN"` setting)

Reply:
Fixed